packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -526,9 +529,11 @@ export class IndexedDbPersistence implements Persistence {
       // the client atomically, but we can't. So we opt to delete the IndexedDb
       // entries first to avoid potentially reviving a zombied client.
       inactiveClients.forEach(inactiveClient => {
-        this.window.localStorage.removeItem(
-          this.zombiedClientLocalStorageKey(inactiveClient.clientId)
-        );
+        if (this.webStorage) {
```

Comment:
nit: I would do this check once on the outside of the for-loop rather than repeat it on every iteration.